doc/Actions.md

```diff
@@ -34,10 +34,22 @@ Remove characters in metadata title fields.
 
 - `pattern` (required): A regular expression to search for. Example: `<<(.*?)>>` would remove "<<" and ">>" from titles.
 
+## onDemandAwtPdfboxSingleFileConvertAction
+
+Converts master images to PDF or JPG using java implementations. Default convert action for the file server.
+
+## onDemandIMSingleFileConvertAction
+
+Converts master images to PDF or JPG using imagemagick / graphicsmagick. Optional convert action for the file server.
+
 ## preproduceDerivativesAction
 
 Generate derivatives for master files in a METS file using Java converter.
 
+## preproduceIMDerivativesAction
```

Comment:
You've cut the parameters from the action above :-) They have the same parameters. I think you should copy it as the user might not recognize that the actions internally are the same with the same parameters.

Reply:
Added parameters